rpc_api.go

```diff
@@ -335,24 +334,6 @@ func (rpcapi *RPCAPI) ConsensusPeers(ctx context.Context, in struct{}, out *[]pe
 	return err
 }
 
-/*
```

Comment:
Should we really be 'just' removing these RPC calls? Yes, we are pre-v1 but I feel like we should provide at least some form of depreciation period. Especially, considering we don't know to what extent our users have integrated with the API.

Reply:
RPC API is internal, so yes, I'm happy 'just' removing.

Reply:
👍